jsapp/scss/components/_kobo.modal.scss

```diff
@@ -698,6 +698,128 @@ $z-modal-x: 10;
   }
 }
 
+.form-media__label {
+  font-weight: 400;
+  font-size: 18px;
+  text-align: center;
+  display: block;
+  margin-top: 20px;
+  margin-bottom: 5px;
+  color: $cool-gray;
+}
+
+.form-media__upload-url {
+  width: 70%;
+  margin: auto;
+  display: block;
+
+  label.form-media__label {
+    text-align: center;
+  }
+
+  // Custom TextBox css
+  label.text-box {
+    min-width: 80%;
+    display: inline-table;
+  }
+
+  button {
+    min-width: 19%;
+  }
+
+
+}
+
+.form-media__list-label {
+  margin-top: 32px;
+  color: black;
+  font-weight: 500;
+  font-size: 22px;
+  display: block;
+  margin-bottom: 11px;
+  border-bottom: 1px solid lighten($cool-gray, 25%);
+  padding-bottom: 16px;
+}
+
+.form-media__default-item {
+  padding-left: 10px;
+}
+
+.form-media__list-item {
+  font-size: 16px;
+  position: relative;
+  padding-bottom: 12px;
+  border-bottom: 1px solid lighten($cool-gray, 25%);
```

Comment:
Please use color from `colors.scss` and try to match some that wouldn't require `lighten` function